XYZDrivers/src/main/java/com/xyzdrivers/models/Claim.java

```diff
@@ -9,21 +9,50 @@
 package com.xyzdrivers.models;
 
 import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
 
 public class Claim extends Model {
 
+    private Integer id;
     private String member_id;
     private LocalDate date;
     private String reason;
     private String status;
     private double amount;
+    
+    private List<String> allowedStatuses;
+    
+    public static String TABLE_NAME = "claims";
+    public static String PRIMARY_KEY = "id";
 
+    public Claim(Integer id, String member_id, LocalDate date, String reason, String status, double amount) {
+        this.id = id;
+        this.member_id = member_id;
+        this.date = date;
+        this.reason = reason;
+        this.status = status;
+        this.amount = amount;
+        this.allowedStatuses = new ArrayList<>();
+        allowedStatuses.add("APPROVED");
+        allowedStatuses.add("REJECTED");
+        allowedStatuses.add("SUBMITTED");
+    }
+    
     public Claim(String member_id, LocalDate date, String reason, String status, double amount) {
         this.member_id = member_id;
         this.date = date;
         this.reason = reason;
         this.status = status;
         this.amount = amount;
+        this.allowedStatuses = new ArrayList<>();
+        allowedStatuses.add("APPROVED");
+        allowedStatuses.add("REJECTED");
+        allowedStatuses.add("PENDING");
```

Comment:
I've approved but don't forget to change this one to submitted as well!

Reply:
damn i think i missed this from the merge

Reply:
good spot